gameplay.md

```diff
@@ -0,0 +1,43 @@
+## Maps
+Given we have a map  of 100 x 100 squares       
```

Comment:
I think that the 100x100 map is a little too big for the beginning. What do you think? What about an initial 10x10 map and then scale up as much as we want? I hope that we will talk more over the next meeting 😃!

Reply:
Great suggestion.